.github/workflows/ci.yml

```
@@ -269,7 +269,7 @@ jobs:
 
     # Skip redundant checks for library releases
     # Skip if run on a fork
```

I'd mention here that access to secrets is required, as in the line diff above.
```suggestion
    # Skip if run without access to secrets (e.g. a fork)
```